staff/victor-abadia/cv/style.css

```diff
@@ -0,0 +1,43 @@
+body {
+    color: #313131;
+    background-color: skyblue;
+    border: 4px solid black;
+    margin: 55px;
+}
+a:link {
+    color: red
+}
+a:visited {
+    color: rgb(216, 3, 3);
+}
+h2 {
+    color: brown;
+    text-align: center;
+  }
+.container {
+    display: flex;
+}
+section {
+    margin: 20px;
+    border: 2px solid black;
+    border-radius: 25px;
+    overflow: auto;
+    padding: 7px;
+    padding-bottom: 20px;
+}
+.navigate {
+    border: 2px solid rgb(146, 13, 13);
+    margin: 20px;
+}
+.box{
+    border-color:blue;
+}
+#footer  {
+    text-align: center;
+}
+flexbox
+@media (max-width: 600px) {

```

Comment:
your rule (media) is projected as desktop-first, that is, it is fired when width is less-or-equal to max-width. the challenge is to make it the inverse: mobile-first. arrange styles to make it fully that, mobile-first.